docs/porting-guides/6.x.x.md

```diff
@@ -123,8 +123,16 @@ The following data model keys have been removed from `eos_cli_config_gen` in AVD
 | `router_bgp.vrfs[].address_family_ipv4_multicast.redistribute_routes` | `router_bgp.vrfs[].address_family_ipv4_multicast.redistribute` |
 | `router_bgp.vrfs[].address_family_ipv6_multicast.redistribute_routes` | `router_bgp.vrfs[].address_family_ipv6_multicast.redistribute` |
 | `router_path_selection.tcp_mss_ceiling.ipv4_segment_size` | `router_path_selection.tcp_mss_ceiling.ipv4` |
+| `static_routes[].destination_address_prefix` | `static_routes[].prefix` |
+| `static_routes[].gateway` | `static_routes[].next_hop` |
+| `ipv6_static_routes[].destination_address_prefix` | `ipv6_static_routes[].prefix` |
+| `ipv6_static_routes[].gateway` | `ipv6_static_routes[].next_hop` |
 | `vlan_interfaces[].vrrp_ids[].ipv6.address` | `vlan_interfaces[].vrrp_ids[].ipv6.addresses` |
 
+### `static_routes.prefix` and `ipv6_static_routes.prefix` are now required keys
```

Comment:
I will remove this as the schema is enough to understand. We may end up doing some table at the end about "new schema restrictions"